.golangci.yml

```diff
@@ -33,3 +33,12 @@ run:
   skip-dirs:
     - pkg/mocks
     - test
+
+issues:
+  # Excluding configuration per-path, per-linter, per-text and per-source
+  exclude-rules:
+    # Exclude some linters from running on tests files.
+    - path: _test\.go
+      linters:
+        - dupl
+        - gosec
```

Comment:
Go sec was complaining about exec-ing a local script in a test. Just removed it from linting our test code.